arcade/gui/constructs.py

```diff
@@ -72,3 +73,76 @@ def on_ok(self, event):
         self.parent.remove(self)
         if self._callback is not None:
             self._callback(event.source.text)
+
+
+class UIDisappearingInfoBox(UIMouseFilterMixin, UIAnchorLayout):
+    """
+    Represents a simple dialog box that pops up with a message and disappears after a
+    certain amount of time.
+
+    Parameters
+    ----------
+    width: float
```

Comment:
This style of constructor documentation won't properly render when generating doc.

See this class for an example in the same file:
https://github.com/pythonarcade/arcade/blob/f41ae2666029427940e03d4f2d6d3df839a4efea/arcade/gui/constructs.py#L13-L19

Reply:
Yea, I just caught that error. I've made a fix.